dev/MRTCore/mrt/Core/src/MRM.cpp

```diff
@@ -874,49 +876,20 @@ STDAPI MrmGetFilePathFromName(_In_opt_ PCWSTR filename, _Outptr_ PWSTR* filePath
 {
     *filePath = nullptr;
 
-    wchar_t path[MAX_PATH];
-    DWORD size = ARRAYSIZE(path);
-    DWORD length = GetModuleFileName(nullptr, path, size);
-    DWORD lastError = GetLastError();
-    std::unique_ptr<wchar_t[]> pathAllocated;
+    wil::unique_cotaskmem_string path;
+    RETURN_IF_FAILED((wil::GetModuleFileNameW<wil::unique_cotaskmem_string, 256>(nullptr, path)));
```

Comment:
You can leave off the `, 256` parameter and then remove the additional `( )` on this.

Reply:
256 is what i want to override. the default is 128. Many times, the path created by the reunion template is fairly long. Thus i want to bump up the default.